test/OpenTelemetry.Tests/Metrics/MetricSnapshotTests.cs

```diff
@@ -163,5 +163,86 @@ public void VerifySnapshot_Histogram()
             Assert.Equal(2, snapshot2.MetricPoints[0].GetHistogramCount());
             Assert.Equal(15, snapshot2.MetricPoints[0].GetHistogramSum());
         }
+
+        [Fact]
+        public void VerifySnapshot_ExponentialHistogram()
+        {
+            var expectedHistogram = new Base2ExponentialBucketHistogram();
+            var exportedMetrics = new List<Metric>();
+            var exportedSnapshots = new List<MetricSnapshot>();
+
+            using var meter = new Meter(Utils.GetCurrentMethodName());
+            var histogram = meter.CreateHistogram<int>("histogram");
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddMeter(meter.Name)
+                .AddView("histogram", new Base2ExponentialBucketHistogramConfiguration())
+                .AddInMemoryExporter(exportedMetrics)
+                .AddInMemoryExporter(exportedSnapshots)
+                .Build();
+
+            // FIRST EXPORT
+            expectedHistogram.Record(10);
+            histogram.Record(10);
+            meterProvider.ForceFlush();
+
+            // Verify Metric 1
+            Assert.Single(exportedMetrics);
+            var metric1 = exportedMetrics[0];
+            var metricPoints1Enumerator = metric1.GetMetricPoints().GetEnumerator();
+            Assert.True(metricPoints1Enumerator.MoveNext());
+            ref readonly var metricPoint1 = ref metricPoints1Enumerator.Current;
+            Assert.Equal(1, metricPoint1.GetHistogramCount());
+            Assert.Equal(10, metricPoint1.GetHistogramSum());
```

Comment:
Test for min and max as well.

Reply:
It's probably better to do it in `AssertExponentialBucketsAreCorrect` method.

Reply:
Yes, should probably will take care of this on this PR. I noted a bug on the explicit bounds histograms where min/max is actually not captured currently in the snapshot, was gonna fix in another PR, but might just do it here since probably small.

Reply:
Min/max is now tested... gotta take off for the evening, so will fix explicit in a follow up